src/providers/info.rs

```diff
@@ -90,8 +92,8 @@ pub trait InfoProvider {
     async fn safe_info(&self, safe: &str) -> ApiResult<SafeInfo>;
     async fn token_info(&self, token: &str) -> ApiResult<TokenInfo>;
     async fn safe_app_info(&self, url: &str) -> ApiResult<SafeAppInfo>;
-    async fn contract_info(&self, address: &str) -> ApiResult<AddressInfo>;
-    async fn full_address_info_search(&self, address: &str) -> ApiResult<AddressInfo>;
+    async fn add_address_info_from_any_source(&self, address: &str) -> ApiResult<AddressEx>;
+    async fn add_address_info_from_contract_info(&self, address: &str) -> ApiResult<AddressEx>;
```

Comment:
the prefix `add_` throws me off a bit. Maybe:
- `address_ex_from_any_source`
- `address_ex_from_contracts`

Reply:
I would add the `to_` in fron of it